CHANGELOG.md

```diff
@@ -47,6 +47,8 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 
 ### Removed
 
+- [#7014](https://github.com/thanos-io/thanos/pull/7014) *: *breaking :warning:* Removed experimental query pushdown feature to simplify query path. This feature has had high complexity for too little benefits.
```

Comment:
Should we mention that we are working on this in the promql-engine engine? I mean the distributed mode?

Reply:
Ill add that the responsibility for this is in the new thanos engine in distributed mode, but i dont feel great about calling it a hack in the changelog!